src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -72,3 +72,62 @@ def put(self, request, project, filter_id):
         )
 
         return Response(status=201)
+
+    # def put(self, request, project, filter_id):
```

Comment:
Why is all this code commented out?

Reply:
Looks like this PR is based on an outdated master, we'll rebase it.